src/client/styles/styles.less

```diff
@@ -35,7 +35,43 @@ body > section {
 #navbar ul li:hover {
     background-color: #ccc;
 }
+.currneLink{
+    text-decoration: underline !important;
+}
+.container .control-label {
+    text-align: left;
+}
+/*secondaryNavigation begin*/
+.secondaryNavigation {
+    padding-left: 1em;
+    margin: 0;
+}
+.secondaryNavigation li {
+    list-style: none;
+    display: inline-block;
+    margin-right: 10px;
 
+}
+.secondaryNavigation li.topRow a {
+    color: #000000;
+}
+.marginBottom {
+    margin-bottom: 5px;
+}
+.marginTop {
+    display: block;
+    height: 45px;
+}
+.marginRight {
+    margin-right: 5px;
+}
+.fontWeight {
+    font-weight: bold;
+}
+.fontWeight.collapsed {
```

Comment:
styles should not use camelCase. use dash instead, like `.margin-top` instead of `.marginTop`